testkit/gradle-testkit-support/src/main/kotlin/com/autonomousapps/kit/gradle/Plugin.kt

```diff
@@ -10,8 +10,9 @@ public class Plugin @JvmOverloads constructor(
 ) : Element.Line {
 
   override fun render(scribe: Scribe): String = scribe.line { s ->
-    s.append("id ")
+    s.append("id(")
     s.appendQuoted(id)
+    s.append(")")
```

Comment:
I'm not going to be able to accept this PR as-is. The whole point of `Scribe` is that you pass in either GROOVY or KOTLIN in its constructor and then it uses that to modify its behavior. Support for Kotlin DSL is simply not ready yet. If you want to attempt to do that in my intended way, I'm open to that.

Reply:
I completely understand! My PR intent was to get most (not all) of the current rendering functionality of Kotlin into a sort of minimal viable product state with minimal changes to the current infrastructure to just get Kotlin projects building at least. 

Based on your feedback, I guess my question is whether you want Groovy Gradle to look like how you currently have it being rendered or if you're ok with Groovy Gradle rendering in a form that also works with Kotlin Gradle (in many cases)?

I can attempt to make updates that would make the generated code between DslKinds more similar, but also understand if you'd rather take that on since it's probably a larger change.

Reply:
I don't want consumers of `Scribe` to have to care about this. Scribe itself should handle it. That likely would involve enhancements to Scribe -- thinks like `methodCall` which would emit parens for Kotlin and no parens for Groovy. And yeah -- I want the Groovy to look idiomatic.

Why do you _need_ Kotlin projects building? This project generates test fixtures, not "real" production projects. Note that I also want support for Kotlin DSL, but just haven't had time to work on it. I'm curious about your use-case. I appreciate the PR for discussion purposes.

Reply:
Ah! Something like `methodCall` definitely makes sense. I don't have a strong use case for using the Kotlin DSL format here. It's primarily what I've been comfortable working with. So far, I have just been using this to generate projects for test purposes, much like you have in this repository. I just chose to use the Kotlin DSL since I saw it was available. 

I kind of figured that you might want Groovy and Kotlin to take on idiomatic forms. 

Given your feedback I'll see what I can think through for updating `Scribe` when I have some time. I think the important piece that you touched on was to add methods for creating abstract language constructs like `methodCall`.

Reply:
I might take this up myself soon. I very much appreciate the interest and the strong signal that being able to write out Kotlin DSL is desirable 👍

Reply:
Sweet! Looking forward to it! For now, I'll fall back to letting everything get generated using Groovy Gradle in my project instead.